.babelrc

```diff
@@ -1,3 +1,13 @@
 {
-  "presets": ["@babel/preset-env"]
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": [
+          "defaults",
+          "IE 11"
+        ]
```

Comment:
Rather than define this here I think it would be better defined in package.json:

```
  "browserslist": "defaults, IE 11"
```

As you've probably seen on Slack I've found myself quite baffled as to how Babel behaves with choosing the browserslist here and inconsistent behaviour.

Life seems to be simpler if we have it in the package.json file, since it's the top source on: https://github.com/browserslist/browserslist/blob/main/README.md#queries and will allow looking up supported browsers with `npx browserslist`.

Also perhaps for a second commit. Ollie suggests we can probably get rid of this babelrc file and move it into package.json under a babel key e.g:

```
  "babel": {
    "presets": ["@babel/preset-env"]
  },
  "browserslist": "defaults",
```

I tried it myself and it seemed to work without problem.

Also commit message here could use an update. Babel hasn't dropped IE11 support it's that browserslist default no-longer includes it. On GOV.UK we specifically continue to support IE11 (for now) and thus need to expand the browsers Babel targets manually.

Reply:
Updated